apps/backend/src/modules/categories/category.service.ts

```diff
@@ -70,27 +81,37 @@ class CategoryService implements Service {
 		const categoryEntity = await this.categoryRepository.find(id);
 
 		return categoryEntity
-			? this.convertCategoryEntityToDto(categoryEntity)
+			? this.parseEntityToCategoryDto(categoryEntity)
 			: null;
 	}
 
-	public async findAll(): Promise<{ items: CategoryDto[] }> {
+	public async findAll(): Promise<GetCategoriesDto> {
+		const entities = await this.categoryRepository.findAll();
+
+		return {
+			items: entities.map((entity) => this.parseEntityToCategoryDto(entity)),
+		};
+	}
+
+	public async findAllWithScores(): Promise<{
+		items: CategoryWithScoresDto[];
+	}> {
 		const categories = await this.categoryRepository.findAll();
 
 		const items = categories.map((categoryEntity) => {
-			return this.convertCategoryEntityToDto(categoryEntity);
+			return this.parseEntityToCategoryWithScoresDto(categoryEntity);
```

Comment:
Is it possible to figure out a shorter name?

Reply:
Will try move the parse logic into the entity 👍🏽